apify-api/openapi/paths/request-queues/request-queues@{queueId}@requests@unlock.yaml

```diff
@@ -0,0 +1,55 @@
+post:
+  tags:
+    - Storage/Request queues/Requests locks
+  summary: Unlock requests
+  description: |
+    Unlocks requests in the queue for the client. The requests are unlocked if were locked by the same client key or if locked from the same Actor run.
+  operationId: requestQueue_requests_unlock_post
+  parameters:
+    - name: queueId
+      in: path
+      description: Queue ID or `username~queue-name`.
+      required: true
+      style: simple
+      schema:
+        type: string
+        example: WkzbQMuFYuamGv3YF
+    - name: clientKey
+      in: query
+      description: |
+        A unique identifier of the client accessing the request queue. It must
+        be a string between 1 and 32 characters long
```

Comment:
I'd use `minLength` and `maxLength` instead (https://swagger.io/docs/specification/v3_0/data-models/data-types/#strings)

Reply:
Okay, are you up to it or shall I make an issue? :slightly_smiling_face: